### DISCUSSION TITLE
Erkennung von Buchstaben + und -

### DISCUSSION THREAD

**Discussion by Fireblade900rr** [jomjol/AI-on-the-edge-device#2269] [Ideas]
Gibt es eine Möglichkeit statt nur Zahlen auch Buchstaben oder Rechenzeichen zu erkennen ?

Hintergrund ist folgender, ich möchte einen digitalen Stromzähler auslesen der aber bei Einspeisung ins Netz leider auch eine positive Wirkleistung anzeigt.

Er zeigt allerdings auch ein großes A mit - oder + daneben an, je nach Verbrauch/Einspeisung, dieses könnte ich verwenden um das zu unterscheiden. Wichtig dabei wäre, dass wenn nichts erkannt wird ein definierter Wert oder NaN ausgeben wird.

Ist das kleinste Intervall zur Verarbeitung 3 Minuten ?

ehz-iw8e2a5l0ek2p

![image](https://user-images.githubusercontent.com/56742771/228902399-99deb647-4f54-4716-bb9c-7d872d0f39b7.png)

Wieso wird hier trotz gar keiner Anzeige eine 1 erkannt ?

![image](https://user-images.githubusercontent.com/56742771/229072446-2c1a5efa-34ca-44b5-80c8-0557b002010c.png)

**Comment by wg25** [comment]
Kann so ein moderner Zähler nicht zuverlässiger mit der integrierten Schnittstelle ausgelesen werden?

**Reply by Fireblade900rr** [reply]
Tja, das wäre der optimalfall, ich konnte in den Daten die er liefert aber kein Flag oder sonst etwas entdecken das die Einspeisung signalisiert. 

OBIS (hex) | OBIS | Name | Wert | Einheit | Parsed
-- | -- | -- | -- | -- | --
0x0100000009ff | 0.0.9 | Geräteeinzelidentifikation | 080534d12dxxxxxx | Unbekannte Einheit | 080534d12xxxxxxUnbekannte Einheit (Geräteeinzelidentifikation)
0x0100010800ff | 1.8.0 | Zählerstand Total | xxxxx3052 | Wh | xxxxx305.2Wh (Zählerstand Total)
0x0100010801ff | 1.8.1 | Zählerstand Tarif 1 | xxxxx3052 | Wh | xxxxx305.2Wh (Zählerstand Tarif 1)
0x0100010802ff | 1.8.2 | Zählerstand Tarif 2 | 0 | Wh | 0.0Wh (Zählerstand Tarif 2)
0x01000f0700ff | 15.7.0 | Unbekannter Datentyp | 4749 | W | 474.9W (Unbekannter Datentyp)

**Comment by caco3** [comment]
Das wird zur Zeit nicht unterstützt.

Siehe auch https://github.com/jomjol/AI-on-the-edge-device/issues/1936 und https://github.com/jomjol/AI-on-the-edge-device/issues/2177
